src/webpack/plugin/SitePlugin.js

```diff
@@ -59,128 +59,145 @@ class SitePlugin {
     sourceFolder,
     distributionFolder,
     basename,
+    isWebpack5,
   }) {
     if (oneTimeSetupComplete) {
       return;
     }
     oneTimeSetupComplete = true;
+    const mdxLoader = getMdxLoader(compiler.options.output.publicPath);
 
-    compiler.options.entry = {
-      ...compiler.options.entry,
-      rewriteHistory: '@cerner/terra-dev-site/lib/browser-router-redirect/rewriteHistory',
-      redirect: '@cerner/terra-dev-site/lib/browser-router-redirect/redirect',
-    };
+    // If a mono repo, update the rootDirectories to include all the packages.
+    const rootDirectories = [
```

Comment:
I personally like checking for lerna.json rather than packages. We could actually set up lerna to not use a packages directory.

Reply:
I am checking for lerna.json:
https://github.com/cerner/terra-dev-site/blob/Support-webpack5/src/webpack/plugin/SitePlugin.js#L16

Reply:
but i do assume packages instead of checking lerna json for what the directory is... we could fix that if it ever becomes a problem.